.github/workflows/terraform-plan-apply.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       is_empty_matrix: ${{ steps.set-matrix.outputs.is_empty_matrix }}
     steps:
       - uses: actions/checkout@v2
+      - uses: chrisdickinson/setup-yq@latest
```

Comment:
Is this the latest version of YQ? It doesn't look like it... also, the syntax of the command is a bit weird...
Are you sure we should use it (hardly any reviews on the action)?

Reply:
Fixed, after the fix didn't change jq syntax besides replacing -r to eval